.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/smartdevicelink/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
+	url = https://github.com/LuxoftSDL/rpc_spec.git
```

Comment:
In fact you don't need this changes, your commit with link to rpc spec will be used automatically. So please delete this file from diff.